core/metacling/src/TCling.cxx

```diff
@@ -2716,6 +2716,14 @@ void TCling::InspectMembers(TMemberInspector& insp, const void* obj,
       return;
    }
 
+   if (TClassEdit::IsUniquePtr(cl->GetName())) {
+      // Ignore error caused by the inside of std::unique_ptr
```

Comment:
Since this is a crucial part of the mechanism, I would still like to see a bit more explanation here. I can imagine at least:
1. Adding some reference to where this `isTransient` flag is used within this function or to which functions it is passed where it becomes important for the logic. By quickly looking at the code, I see it's used in the calls to `InspectMember`.
2. A reference to the GitHub issue that triggered this change.

Reply:
There is 81 uses of `isTransient` throughout `core and an additional 120 uses of the related `silent`.
The semantic of these 2 is essential `please do your thing but don't print out any complaint if something is odd because it is not for persistency and thus not important`

Reply:
Understood, thanks!